src/helpers/callbacks.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Nectary\Helpers;
+
+use Nectary\Response;
+
+trait Callbacks {
+  public static function error() {
```

Comment:
What about `not_found_error()` instead?
We could also want to return a few other HTTP responses in the future, like a `401` or `403` which are also errors.